.devcontainer/devcontainer.json

```diff
@@ -1,19 +1,62 @@
+// For format details, see https://aka.ms/devcontainer.json. For config options, see the README at:
+// https://github.com/microsoft/vscode-dev-containers/tree/v0.234.0/containers/dotnet
 {
-  "name": "Octokit.NET",
+  "name": "C# (.NET)",
```

Comment:
[non blocking] Just curious, why the name change here?

Reply:
Just adopting the defaults from the official template - I'll switch this back though as it's a bit nicer.